sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/Rules/DoNotNestTernaryOperatorsTest.cs

```diff
@@ -0,0 +1,38 @@
+﻿/*
+ * SonarAnalyzer for .NET
+ * Copyright (C) 2015-2018 SonarSource SA
+ * mailto: contact AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+extern alias csharp;
```

Comment:
Please do not use the `extern alias csharp` when it is not mandatory.

Reply:
It is, otherwise VS complains (note now all tests have it).

Reply:
I can see that, when was it done? I cannot see the commit (on master).

Reply:
It's 04e700dfe
https://github.com/SonarSource/sonar-csharp/pull/1099/commits